torch/csrc/jit/serialization/import_source.cpp

```diff
@@ -485,13 +485,13 @@ struct SourceImporterImpl : public Resolver,
         } break;
         case TK_DEF: {
           Def def = Def(statement);
-          if (pre_hook_names.find(def.name().name()) != pre_hook_names.end()) {
-            pre_hook_def_map.emplace(def.name().name(), def);
-            pre_hook_resolver_map.emplace(
-                def.name().name(), shared_from_this());
-          } else if (hook_names.find(def.name().name()) != hook_names.end()) {
-            hook_def_map.emplace(def.name().name(), def);
-            hook_resolver_map.emplace(def.name().name(), shared_from_this());
+          auto def_name = def.name().name();
+          if (pre_hook_names.find(def_name) != pre_hook_names.end()) {
+            pre_hook_def_map.emplace(def_name, def);
+            pre_hook_resolver_map.emplace(def_name, shared_from_this());
+          } else if (hook_names.find(def_name) != hook_names.end()) {
+            hook_def_map.emplace(def_name, def);
+            hook_resolver_map.emplace(def_name, shared_from_this());
```

Comment:
This looks like a good clean up, but it is not related to this named tuple functionality in this PR, please make a separate PR for it.

Reply:
Sure, reverted